index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>portofolio: setup and mobile version skeleton</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body class="">
+    <header class="navibar">
+      <nav class="nav-container">
+        <a class="logo" href="#">My Logo</a>
+        <img class="mobile-menu" src="img/Union.png" alt=" navigation" />
+      </nav>
```

Comment:
- [ OPTIONAL] Please make sure that the menu is on the right side and it looks exactly like Figma design 

![image](https://user-images.githubusercontent.com/74744787/159786388-4c04458d-21a8-4877-81c8-546d1aeb5448.png)